compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1577,7 +1577,46 @@ OMR::ARM64::TreeEvaluator::vdsqrtEvaluator(TR::Node *node, TR::CodeGenerator *cg
 TR::Register*
 OMR::ARM64::TreeEvaluator::vfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild = node->getThirdChild();
+
+   TR::Register *thirdReg  = cg->evaluate(thirdChild);
+   TR::Register *secondReg  = cg->evaluate(secondChild);
+   TR::Register *firstReg  = cg->evaluate(firstChild);
```

Comment:
Is there any reason for evaluating the child nodes in the order from third to first?

Reply:
No. I think we can do it from first to third for simplicity.